pandas/io/parsers.py

```diff
@@ -1065,6 +1065,20 @@ def _evaluate_usecols(usecols, names):
     return usecols
 
 
+def _validate_usecols(usecols, names):
+    """
+    Validates that all usecols are present in a given
+    list of names, if not, raise a ValueError that
```

Comment:
Nit: let's remove the comma splice in this doc-string i.e. 

"...names. If not, raise a ValueError that..."

Reply:
Also, let's add a parameter description + return `usecols` (just in case).

Reply:
Done & done!

I've used the doc format of the existing functions that were below, if this needs to change let me know and I'll do so.